src/pages/index.js

```diff
@@ -2,97 +2,227 @@ import "./index.css"; // add import of the main stylesheets file
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import {
+  avatarButton,
   validationSettings,
-  initialCards,
-  editProfileForm,
-  addCardForm,
-  cardListElement,
-  editProfileModal,
-  addCardModal,
-  previewImageModal,
-  editProfileButton,
+  profileNameInput,
+  profileDescriptionInput,
   addCardButton,
-  profileName,
-  profileDescription,
-  cardSelector,
+  editProfileButton,
+  selectors,
+  apiInfo,
 } from "../utils/constants.js";
 import Section from "../components/Section.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import UserInfo from "../components/UserInfo.js";
+import Api from "../components/Api.js";
 
-// Initial start-up
+// Message to Tripleten reviewer: I'm so sorry for the terrible code.
+// I have really been struggling with this project and I dont know what I'm doing wrong
+// I keep getting an error in Popup.js, but I don't know whats wrong with it
+// Thank you for your patience
```

Comment:
Dear Timothy , (**Please, expand the general comment ↓**)

Unfortunately I have to reject the project:



* There are errors and nothing works https://disk.yandex.ru/i/NtwMq8n-QTDRuQ  ` super({ selector });` causes errors in `Popup` because you pass **an object** but you need to pass a **string**. So, it should be `super(selector);`